services/tasks-svc/internal/task/commands/v1/update_task.go

```diff
@@ -50,6 +36,18 @@ func NewUpdateTaskCommandHandler(as hwes.AggregateStore, authz hwauthz.AuthZ) Up
 			}
 		}
 
+		if public != nil && a.Task.Public != *public {
+			if err := a.UpdateTaskPublic(ctx, *public); err != nil {
+				return err
+			}
+		}
+
+		if dueAt != nil {
+			if err := a.UpdateDueAt(ctx, dueAt.AsTime()); err != nil {
+				return err
+			}
+		}
```

Comment:
then it should be removed, right?

Reply:
We have an extra endpoint & command for removing the `DueAt` (`RemoveTaskDueAt`).